hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/util/SignalLogger.java

```diff
@@ -56,9 +65,8 @@ private static class Handler implements SignalHandler {
      * @param signal    The incoming signal
      */
     @Override
-    public void handle(Signal signal) {
-      log.error("RECEIVED SIGNAL " + signal.getNumber() +
-          ": SIG" + signal.getName());
+    public void handle(int signal) {
+      log.error("RECEIVED SIGNAL {}: SIG{}", signal, Signal.valueOf(signal));
```

Comment:
`Signal` name includes `SIG...`, so I think this would log e.g. `SIGSIGHUP`.

```suggestion
      log.error("RECEIVED SIGNAL {}: {}", signal, Signal.valueOf(signal));
```

Reply:
Thanks, you are right, I have fixed it.